satellite/metainfo/config.go

```diff
@@ -21,7 +21,7 @@ const (
 // RSConfig is a configuration struct that keeps details about default
 // redundancy strategy information
 type RSConfig struct {
-	MaxSegmentSize   memory.Size `help:"maximum segment size" default:"64MiB"`
+	MaxSegmentSize   memory.Size `help:"maximum segment size" default:"128MiB"`
```

Comment:
why are we changing this ?

Reply:
Uplink's limit is set to 64MiB(unencrypted segment size), when max limit segment size is encrypted, it goes beyond 64MiB which is also the default max limit set on the satellite side as well. There is a condition to check on the satellite side to make sure the segment size to be less than max default limit. Unless those defaults are changed on the config.yaml, the storj-sim test to upload a 500MB file is failing due to the size check. I changed the default on the satellite to be twice the segment size, that way we can test large files on development environment.